.github/workflows/assign-issue-to-pipeling-issue-tracker.yml

```diff
@@ -0,0 +1,22 @@
+name: Assign Issue to Pipeling Issue Tracker
+
+on:
+  workflow_call:
+    inputs:
+      issue_number:
+        required: true
+        type: number
+      repository:
+        required: true
+        type: string
+
+jobs:
+  add-to-project:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add issue to pipeling issue tracker project
+        uses: actions/add-to-project@v1.0.2
```

Comment:
pin to sha
e.g - https://github.com/turbot/steampipe-workflows/blob/main/.github/workflows/registry-publish-ghcr.yml#L42